src/Adapter/ContainerBuilder.php

```diff
@@ -170,7 +170,7 @@ private function compileContainer()
         //If the container builder is modified the container logically should be rebuilt
         $container->addResource(new FileResource(__FILE__));
 
-        $container->addCompilerPass(new LoadServicesFromModulesPass($this->containerName), PassConfig::TYPE_BEFORE_OPTIMIZATION, 1);
+        $container->addCompilerPass(new LoadServicesFromModulesPass($this->containerName), PassConfig::TYPE_BEFORE_OPTIMIZATION, 200);
```

Comment:
Can you please add a comment here explaining why this specific priority is used, that it needs to have a higher priority than `ResolveClassPass` and `ResolveInstanceofConditionalsPass` (too bad this priority is hard-coded in the framework a const would have been nice to use as a reference)

Reply:
Thanks for the suggestion. The CS fixer did not like the phpDoc comment above the method call (because of the `phpdoc_to_comment` rule), so I added a new constant in the `PrestaShopBundle` class, added a comment there and replaced the hard-coded 200 priorities with the constant usage.